exercises/two-fer/src/lib.rs

```diff
@@ -1,3 +1,6 @@
 pub fn twofer(name: &str) -> String {
-    unimplemented!("One for {}, one for me.", name);
+    match name {
```

Comment:
> although it does not hurt to have it since it's deprecated, this lib.rs doesn't necessarily need to have a complete implementation (and we would want it not to have one if it were non-deprecated)

same